.env.example

```diff
@@ -17,4 +17,7 @@ GOOGLE_CLIENT_ID=
 GOOGLE_CLIENT_SECRET=
 
 GITHUB_CLIENT_ID=
-GITHUB_CLIENT_SECRET=
\ No newline at end of file
+GITHUB_CLIENT_SECRET=
+
+SECRET=
+EMAIL_VERIFICATION_URL=
```

Comment:
Let's add the SENDGRID API KEY to this env example @ddouglasz

Reply:
Okay. Doing it now. Thank you.